src/tests/components/experiments/ExperimentList.test.js

```diff
@@ -48,3 +56,44 @@ describe("Experiment list component", () => {
     );
   });
 });
+
+describe("Test renderTitle() method", () => {
+  it("should be passed both startDate and endDate as props", () => {
```

Comment:
I don't know if this is the right name for this test, the props are what go in, but the test is testing what comes OUT, so it should describe that.